sql/core/src/main/java/org/apache/spark/sql/sources/v2/SupportsWrite.java

```diff
@@ -22,9 +22,10 @@
 import org.apache.spark.sql.util.CaseInsensitiveStringMap;
 
 /**
- * An internal base interface of mix-in interfaces for writable {@link Table}. This adds
- * {@link #newWriteBuilder(CaseInsensitiveStringMap)} that is used to create a write
- * for batch or streaming.
+ * A mix-in interface of {@link Table}, to indicate that it's readable.
```

Comment:
@jose-torres, @cloud-fan: Maybe the capability/trait distinction would be more clear if this stated "... to indicate that it's readable using the v2 datasource API".

Reply:
We don't have a plan to introduce something similar to the `Table` API, I'd like to be clear right now and say this is a mix-trait for `Table`.

Reply:
That's fine with me. Can we make this change part of a separate documentation update?